config-ui/src/hooks/useWebhookManager.jsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import { useState, useEffect } from "react";
+
+import request from "@/utils/request";
+import { DEVLAKE_ENDPOINT } from "@/utils/config";
+
+export const useWebhookManager = () => {
+  const [loading, setLoading] = useState(false);
+  const [operating, setOperating] = useState(false);
```

Comment:
Can we refactor this to `isFetching` to be consistent with the fetch flags used in other hook services? You may alias it to `operating` or another preferred alias when the hook is implemented.
```suggestion
  const [isFetching, setIsFetching] = useState(false);
```

Reply:
I think operating includes save, update and delete operations, isSaving cannot express this meaning.